doc/source/contribute.rst

```diff
@@ -44,20 +44,24 @@ Creating a new release
 
 New releases follow these steps:
 
-- Create a new branch ``release/YYYY.MM.DD`` based on the ``develop`` branch.
-  - ``git checkout -b release/YYYY.MM.DD develop``
-- Create a Github pull request to merge ``release/YYYY.MM.DD`` into ``master``.
+- Create a new branch ``release-YYYY.MM.DD`` based on the ``develop`` branch.
+  - ``git checkout -b release-YYYY.MM.DD develop``
+- Create a Github pull request to merge ``release-YYYY.MM.DD`` into ``master``.
```

Comment:
I also liked the slash `/`, gitflow has this by default

Reply:
I have no strong feelings about it, I just happened to use a dash when I made the first branch, I think because there's already a p4a branch with a dash. I'll just revert this part if you like.

Reply:
OK I was wondering what was the reason behind. I think we can use whatever the person doing the release like. The idea behind is the same; having a release branch.
So let's keep it the way you like